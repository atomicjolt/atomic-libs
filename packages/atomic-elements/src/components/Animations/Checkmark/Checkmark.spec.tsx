import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { Checkmark } from ".";

describe("Checkmark", () => {
  it("should match snapshot", () => {
    const res = render(<Checkmark />);
    expect(res.container).toMatchSnapshot();
  });
});
