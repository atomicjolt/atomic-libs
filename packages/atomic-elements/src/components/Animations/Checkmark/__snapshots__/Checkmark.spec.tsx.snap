// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Checkmark > should match snapshot 1`] = `
<div>
  <span
    class="sc-blHHSb MEcYg aje-checkmark is-medium"
    data-testid="checkmark"
  />
</div>
`;
